package.json

```diff
@@ -42,9 +42,8 @@
   },
   "scripts": {
     "adapter": "truffle exec scripts/adapter_scripts/run.js --network rinkeby",
-    "deploy": "truffle migrate --network rpc",
-    "deploy:rinkeby": "truffle migrate --network rinkeby",
-    "deploy:prepare": "rm -rf build && npm run compile && truffle exec scripts/prepare.js --network",
```

Comment:
is there still a script to deploy the adapters?

Reply:
the `adapter` script you added is still there

Reply:
what if we update the code? or add a new dependencies (like dutchX adapter)? Do we still want a deploy_dep type thing?

Reply:
I think we will handle this one differently than the deploy_dep in futarchy-app, mostly because that was designed for local and testnet/mainnet, and I think for this app the local config will be different (uses a mock contract and doesn't have the same dependencies)